modules/ROOT/pages/functions/templates/node.adoc

```diff
@@ -0,0 +1,267 @@
+Node.js Function Template Reference
+===================================
+
+The Node.js function templates are designed to allow developers to concentrate
+solely on the business logic of the application, leaving things like managing
+network connections to the runtime framework. Whether responding to
+`CloudEvents` or simple HTTP requests, the function templates are simple and
+straightforward.
```

Comment:
It would be better if this were more specific about all the things that are actually managed by the runtime network, it's a bit vague / hard to understand what a user actually gains from using this.